Automattic/wc-calypso-bridge
# Update Plugins Landing Page Copy and Illustration

### Changes proposed in this Pull Request:



This PR delivers the newest changes on the Plugins landing page on free trial stores.

Screenshot of the result:
<img width="1850" alt="Screenshot 2023-09-13 at 7 37 40 PM" src="https://github.com/Automattic/wc-calypso-bridge/assets/1105590/ecf53990-31c0-40e8-befe-695b17454764">

Closes #1299 






### How to test the changes in this Pull Request:




1. Apply this patch on a free trial store
2. Navigate to "Plugins"
3. Notice the new design




### Events 

This PR introduces the `free_trial_hire_an_expert` event and deprecates the `free_trial_get_inspired` event.
